app/views/dashboard/_tab_normalized_data.html.erb

```diff
@@ -1,3 +1,43 @@
 <div class="tab-pane fade" id="normalized-data-pane" role="tabpanel" aria-labelledby="normalized-data-tab">
-    Normalized data here
-</div>
+    <table class="table table-striped mb-3" id="normalized-data-table">
+      <thead>
+        <tr>
+          <th><%= t('dashboard.summary.table.provider') %></th>
+          <th><%= t('dashboard.summary.table.last_full_dump') %></th>
+          <th class="text-end pe-4"><%= t('dashboard.summary.table.dump_records') %></th>
+          <th><%= t('dashboard.summary.table.last_delta') %></th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @dashboard.normalized_data_by_provider.sort_by{ |k, _v| _v.updated_at}.reverse.each do |provider, dump| %>
```

Comment:
This doesn't account for the possibility that `_v` can be nil.

The variable name prefixed with an underscore signals that it is not used, so I think you want `sort_by { |_k, v| v.updated_at }`

Reply:
@corylown my bad. Thanks.

Reply:
@corylown lol I rolled out `Time.at(0)` in my attempt to solve this nil issue in latest commit, not sure of another better way!